images/fcos-bastion-image/root/usr/bin/dhcp-init.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+set -x
+
+mkdir -p /var/opt/{ignitions,html}
+cp ${LOCAL_PATH}/dhcp/ignitions/* /var/opt/ignitions/
+
+ign=/var/opt/ignitions
+mkdir -p ${PODMAN_DNSMASQ_BASEDIR}/{etc,tftpboot,misc}
+#/usr/bin/chown -R 65534:65534 ${PODMAN_DNSMASQ_BASEDIR}/misc
+cp ${LOCAL_PATH}/dhcp/bastion_dnsmasq.conf ${PODMAN_DNSMASQ_BASEDIR}/etc/
```

Comment:
NIT: can we call this file with the default name (dnsmasq.conf)?

Reply:
done